### PR TITLE
[FIX] asv

### PR DESCRIPTION
**Pull request by edouardoyallon**

### CODE REVIEW

File: benchmarks/asv.conf.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     // uninstalling the project. See asv.conf.json documentation.
     //
     "install_command": [
-        "pip install torch torchvision",
+       // "conda install pytorch torchvision torchaudio",
```
Comment (lostanlen) [inline benchmarks/asv.conf.json:20]: remove this line completely. let's focus on the core dependency (numpy) for the time being

File: benchmarks/asv.conf.json [hunk 2]
```diff
@@ -45,11 +45,11 @@
 
     // The Pythons you'd like to test against.  If not provided, defaults
     // to the current version of Python used to run `asv`.
-    "pythons": ["3.7"],
+    "pythons": ["3.8"],
```
Comment (lostanlen) [inline benchmarks/asv.conf.json:48]: 3.7 works on my machine.
(3.6 doesn't though)

File: benchmarks/benchmarks/scattering1d.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import torch
-import kymatio.scattering1d.backend as backend
+#import torch
+#import kymatio.scattering1d.backend as backend
```
Comment (lostanlen) [inline benchmarks/benchmarks/scattering1d.py:2]: from kymatio.numpy import Scattering1D

File: benchmarks/benchmarks/common.py [hunk 4]
```diff
@@ -1,2 +1,2 @@
-import torch
-torch.manual_seed(0)
+#import torch
+#torch.manual_seed(0)
```
Comment (lostanlen) [inline benchmarks/benchmarks/common.py:2]: set numpy random seed

File: benchmarks/benchmarks/scattering2d.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
-import torch
-import kymatio.scattering2d.backend as backend
+#import torch
+#import kymatio.scattering2d.backend as backend
```
Comment (lostanlen) [inline benchmarks/benchmarks/scattering2d.py:2]: from kymatio.numpy import Scattering2D

File: benchmarks/benchmarks/scattering3d.py [hunk 6]
```diff
@@ -1,5 +1,5 @@
-import torch
-import kymatio.scattering3d.backend as backend
+#import torch
+#import kymatio.scattering3d.backend as backend
```
Comment (lostanlen) [inline benchmarks/benchmarks/scattering3d.py:2]: from kymatio.numpy import Scattering3D

File: doc/source/developerguide.rst [hunk 7]
```diff
@@ -48,10 +48,10 @@ If all the tests pass, you may submit your pull request as explained below. A sp
 Checking speed
 ==============
 
-For checking the speed of a given PR, run the ASV benchmarks on various architectures via::
+For checking the speed of a given PR, first install `asv` via ``pip install asv`` and run the ASV benchmarks on various architectures via::
 
     cd benchmarks
-    run asv
+    asv run
```
Comment (lostanlen) [inline doc/source/developerguide.rst:54]: good catch

File: doc/source/developerguide.rst [hunk 8]
```diff
@@ -48,10 +48,10 @@ If all the tests pass, you may submit your pull request as explained below. A sp
 Checking speed
 ==============
 
-For checking the speed of a given PR, run the ASV benchmarks on various architectures via::
+For checking the speed of a given PR, first install `asv` via ``pip install asv`` and run the ASV benchmarks on various architectures via::
```
Comment (lostanlen) [inline doc/source/developerguide.rst:51]: you're right

**Review by lostanlen — CHANGES_REQUESTED** [review]

File: benchmarks/benchmarks/scattering2d.py (hunk 5 above)
Comment (eickenberg) [reply benchmarks/benchmarks/scattering2d.py:2]: why are these lines now commented?
Shouldn't they either be in code or deleted?

**Comment by eickenberg** [comment]
superseded by https://github.com/kymatio/kymatio/pull/658
